engineapi/backups.go

```diff
@@ -86,8 +86,12 @@ func parseOneBackup(output string) (*Backup, error) {
 	return parseBackup(data)
 }
 
-func (b *BackupTarget) ListVolumes() ([]*BackupVolume, error) {
+func (b *BackupTarget) ListVolumes(credential map[string]string) ([]*BackupVolume, error) {
```

Comment:
I just realize we can put credential inside the BackupTarget. Let's refactor it in after this patch gets merged.

Reply:
But why only List and Get get filled in? How about List(), GetBackup() and DeleteBackup()?